src/coreclr/jit/emit.h

```diff
@@ -1547,11 +1547,30 @@ class emitter
         {
             return _idEvexbContext != 0;
         }
-        void idSetEvexbContext()
+
+        void idSetEvexbContext(insOpts instOptions)
```

Comment:
Do we want to preserve the `assert(_idEvexbContext == 0);` to ensure its not being set twice?